packages/elements/src/overlay/elements/overlay.ts

```diff
@@ -621,6 +626,7 @@ export class Overlay extends ResponsiveElement {
   /**
    * Set focus boundary to restrict tabbing. Default's overlay itself.
    * If external focus is required, set to null
+   * @default overlay
```

Comment:
Tricky. Will it look weird if we do `@default this`?

Reply:
At first it show "this" as default value but I changed it to element name. 
I'm not sure as well which is better between 'this' or 'element-name'.

Reply:
For correctness, I think `this` is more correct because the value is itself, not any `ef-overlay`.
Make sense?

Reply:
Agree. Fixed